tests/test_layers.py

```diff
@@ -613,7 +613,7 @@ def test_channel_last(self):
         self.assertTrue(self.run_onnx_runtime('channel_last_input', onnx_model, x, expected))
 
     def _test_keras_model(self, model, model_name='onnx_conversion', rtol=1.e-4, atol=1.e-8, img_size=224):
-        img_path = 'data/elephant.jpg'
+        img_path = os.path.join(os.path.dirname(__file__), 'data', 'elephant.jpg')
```

Comment:
Can you check in this jpeg file as well?

Reply:
It is already checked-in